core/services/relay/evm/binding/batch.go

```diff
@@ -1,4 +1,4 @@
-package evm
+package binding
```

Comment:
batchCaller doesn't really have anything to do with bindgins, it should stay in the evm package

Reply:
Batch doesn't have anything to do with binding, but it used by the bindings and should not call up the package stack. It could be its own package, but right now this is the only usage. To the contrary, I am not sold on the name `binding` for the package because the package isn't just about binding. It is mapping contracts to read functions. Not sure what would be a good alternative name.

Reply:
If it's about mapping contracts to read functions, what do you think about:
- `mapcontractreading`
- `mapcontractread`
- `mapcontractreads`